test/test-agent-stopped.js

```diff
@@ -21,9 +21,14 @@ var http = require('http');
 
 describe('test-agent-stopped', function() {
   var agent;
-  before(function() {
+  before(function(done) {
     delete process.env.GCLOUD_PROJECT;
     agent = require('..').start();
+    // Wait 300ms for agent to fail getting remote project id.
```

Comment:
sgtm. It would be good to add a nock and get rid of the hard-coded time limit however.